spec/support/CurrentSpecReporter.js

```diff
@@ -1,12 +1,16 @@
 // Sets a global variable to the current test spec
 // ex: global.currentSpec.description
 const { performance } = require('perf_hooks');
+const flakyTests = require('./flakyTests.json');
```

Comment:
Could you add a test in which the JSON file does not exist? It should not be a requirement to have the file in order to run the tests. The file should be optional.

Reply:
This file will always exist similarly I could just hardcode the array in this file.

Reply:
I removed the `flakyTest.json` file and just move the array to the spec reporter